yt_dlp/extractor/neteasemusic.py

```diff
@@ -203,91 +156,150 @@ class NetEaseMusicIE(NetEaseMusicBaseIE):
             'timestamp': 691516800,
             'description': 'md5:1ba2f911a2b0aa398479f595224f2141',
+            'subtitles': {'lyric': [{'ext': 'lrc'}]},
+            'duration': 268,
+            'alt_title': '伴唱:现代人乐队 合唱:总政歌舞团',
+            'thumbnail': r're:^http.*\.jpg',
+        },
+    }, {
+        'url': 'http://music.163.com/#/song?id=32102397',
+        'md5': '3e909614ce09b1ccef4a3eb205441190',
+        'info_dict': {
+            'id': '32102397',
+            'ext': 'mp3',
+            'title': 'Bad Blood',
+            'creator': 'Taylor Swift / Kendrick Lamar',
+            'upload_date': '20150516',
+            'timestamp': 1431792000,
+            'description': 'md5:21535156efb73d6d1c355f95616e285a',
+            'subtitles': {'lyric': [{'ext': 'lrc'}]},
+            'duration': 199,
+            'thumbnail': r're:^http.*\.jpg',
+        },
+        'skip': 'Blocked outside Mainland China',
+    }, {
+        'note': 'Has translated name.',
+        'url': 'http://music.163.com/#/song?id=22735043',
+        'info_dict': {
+            'id': '22735043',
+            'ext': 'mp3',
+            'title': '소원을 말해봐 (Genie)',
+            'creator': '少女时代',
+            'upload_date': '20100127',
+            'timestamp': 1264608000,
+            'description': 'md5:79d99cc560e4ca97e0c4d86800ee4184',
+            'subtitles': {'lyric': [{'ext': 'lrc'}]},
+            'duration': 229,
+            'alt_title': '说出愿望吧(Genie)',
+            'thumbnail': r're:^http.*\.jpg',
         },
+        'skip': 'Blocked outside Mainland China',
     }]
 
     def _process_lyrics(self, lyrics_info):
-        original = lyrics_info.get('lrc', {}).get('lyric')
-        translated = lyrics_info.get('tlyric', {}).get('lyric')
+        original = traverse_obj(lyrics_info, ('lrc', 'lyric', {str}))
+        translated = traverse_obj(lyrics_info, ('tlyric', 'lyric', {str}))
+
+        if not original or original == '[99:00.00]纯音乐，请欣赏\n':
+            return None
 
         if not translated:
             return original
 
         lyrics_expr = r'(\[[0-9]{2}:[0-9]{2}\.[0-9]{2,}\])([^\n]+)'
         original_ts_texts = re.findall(lyrics_expr, original)
-        translation_ts_dict = dict(
-            (time_stamp, text) for time_stamp, text in re.findall(lyrics_expr, translated)
-        )
-        lyrics = '\n'.join([
-            '%s%s / %s' % (time_stamp, text, translation_ts_dict.get(time_stamp, ''))
-            for time_stamp, text in original_ts_texts
-        ])
-        return lyrics
+        translation_ts_dict = {
+            timestamp: text for timestamp, text in re.findall(lyrics_expr, translated)
+        }
+
+        for i in range(len(original_ts_texts)):
+            timestamp, text = original_ts_texts[i]
+            if translation_ts_dict.get(timestamp):
+                original_ts_texts[i] = timestamp, f'{text} / {translation_ts_dict[timestamp]}'
+        return '\n'.join([''.join(parts) for parts in original_ts_texts])
```

Comment:
Is there a specific reason this returned only one lyrics instead of both or is that an artifact from previous code? I would imagine something like this as a result so the user can mix match and filter?
```py
            'subtitles': {
                'lyrics-original': [{
                    'data': lyrics_og,
                    'ext': 'lrc',
                }],
                'lyrics': [{
                    'data': lyrics_translated,
                    'ext': 'lrc',
                }]
            }
```

Reply:
Previous code merges translated lyrics (if exsists) into original lyrics and save the merge lyrics into media description, while it didn't provide a subtitle output.
Here I only added a `subtitles` to redirect the merged lyrics text into a separate file. I agree it would be better to have original and translated lyrics in separate files, though I would like to have those as additional files and keep `'lyrics'` for the merged lyrics.
```
            'subtitles': {
                'lyrics': [{
                    'data': merged_or_original,
                    'ext': 'lrc',
                }],
                'lyrics-original': [{
                    'data': traversed_original,
                    'ext': 'lrc',
                }],
                'lyrics-translated': [{
                    'data': traversed_translated,
                    'ext': 'lrc',
                }],
            }
```

Reply:
While this could be done by an external script, if you think it worth it then do this instead yeah. I would want to revise the merging algorithm though, and can either do that before or after your change to return multiple, whatever order youd want most